### PR TITLE
added more instructions in the getting started readme (#235)

### PR DESCRIPTION
**Pull request by erwin1**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -26,9 +26,46 @@ There are two components of this project
 
 ### Getting Started
 
+For now Pyrsia only supports Docker artifacts. Follow these steps to run a Pyrsia node and use it as the registry for all Docker Hub content.
+
 1. Setup rust on your local machine as described in [Rust's getting started guide](https://www.rust-lang.org/learn/get-started)
 2. `cd pyrsia_node`
```
Comment (mgrand) [inline readme.md:32]: Add a step telling people to check out the Pyrsia source from https://github.com/pyrsia/pyrsia.git

File: readme.md [hunk 2]
```diff
@@ -26,9 +26,46 @@ There are two components of this project
...
 3. `cargo run`
+4. **configure Docker** to use Pyrsia, which is running on localhost port 7878\
+    open your Docker daemon settings and add this entry in the root object:
```
Comment (mgrand) [inline readme.md:35]:
```suggestion
    open your Docker daemon settings and add this entry in the root JSON object:
```

File: readme.md [hunk 3]
```diff
@@ -26,9 +26,46 @@ There are two components of this project
...
+    ```
+      "registry-mirrors": ["http://localhost:7878"]
```
Comment (mgrand) [inline readme.md:37]: You should mention before or immediately after this step the problem with using localhost. I followed this instruction, continued to the next step, found a problem and then noticed that there was a notice at the bottom about other platforms.

**Review by mgrand — APPROVED** [review]

File: readme.md (hunk 3 above)
Comment (erwin1) [reply readme.md:41]: Right, I tried to address your remark by reshuffling things

**Review by mgrand — APPROVED** [review]

**Review by johanvos — APPROVED** [review]

**Pull request merged**